opensourceleg/actuators/dephy.py

```diff
@@ -0,0 +1,1739 @@
+        _coeffs = _beta[0]
+
+        self.set_encoder_map(np.polynomial.polynomial.Polynomial(coef=_coeffs))
+
+        np.save(file=f"./{self.tag}_encoder_map.npy", arr=_coeffs)
+        LOGGER.info(
+            msg=f"[{self.__repr__()}] Encoder map saved to './{self.tag}_encoder_map.npy'."
+        )
+
+    def set_motor_torque(self, value: float) -> None:
+        """
+        Sets the motor torque in Nm.
+
+        Args:
+            value (float): The torque to set in Nm.
+        """
+        self.set_motor_current(
+            value / self.MOTOR_CONSTANTS.NM_PER_MILLIAMP,
+        )
+
+    def set_joint_torque(self, value: float) -> None:
+        """
+        Set the joint torque of the joint.
+        This is the torque that is applied to the joint, not the motor.
+
+        Args:
+            value (float): torque in N_m
+        """
+        self.set_motor_torque(value=value / self.gear_ratio)
+
+    @deprecated_with_routing(alternative_func=set_joint_torque)
+    def set_output_torque(self, value: float) -> None:
+        """
+        Set the output torque of the joint.
+        This is the torque that is applied to the joint, not the motor.
+
+        Args:
+            value (float): torque in N_m
+        """
+        self.set_motor_torque(value=value / self.gear_ratio)
+
+    def set_motor_current(
+        self,
+        value=float,
+    ):
+        """
+        Sets the motor current in mA.
+
+        Args:
+            value (float): The current to set in mA.
+        """
+        self.command_motor_current(value=int(value))
+
+    @deprecated_with_routing(alternative_func=set_motor_current)
+    def set_current(self, value: float) -> None:
+        self.command_motor_current(value=int(value))
+
+    def set_motor_voltage(self, value: float) -> None:
+        """
+        Sets the motor voltage in mV.
+
+        Args:
+            value (float): The voltage to set in mV.
+        """
+        self.command_motor_voltage(value=int(value))
+
+    @deprecated_with_routing(alternative_func=set_motor_voltage)
+    def set_voltage(self, value: float) -> None:
+
+        self.command_motor_voltage(value=int(value))
+
+    def set_motor_position(self, value=float) -> None:
+        """
+        Sets the motor position in radians.
+        If in impedance mode, this sets the equilibrium angle in radians.
+
+        Args:
+            value (float): The position to set
+        """
+        self.command_motor_position(
+            value=int(
+                (value + self.motor_zero_position + self.motor_position_offset)
+                / self.MOTOR_CONSTANTS.RAD_PER_COUNT
+            ),
+        )
+
+    def set_position_gains(
+        self,
+        kp: float = DEFAULT_POSITION_GAINS.kp,
+        ki: float = DEFAULT_POSITION_GAINS.ki,
+        kd: float = DEFAULT_POSITION_GAINS.kd,
+        ff: float = DEFAULT_POSITION_GAINS.ff,
+    ) -> None:
+        """
+        Sets the position gains in arbitrary Dephy units.
```

Comment:
I think we should use the `@deprecated_with_routing` decorator on anything that uses arbitrary "dephy count" units and push everyone towards using real engineering units. 
IMO, any `set_*_gains()` method should accept things in real units unless you dig hard to find the dephy arbitrary ones.